src/domains/ecommerce-app/api/ecommerce-checkout/v1/_openapi.json.tpl

```diff
@@ -389,7 +389,16 @@
             },
             "required": true,
             "description": "Transaction ID"
-          }
+          },
+          {
+            "in": "header",
+            "name": "lang",
+            "required": true,
```

Comment:
why required?

Reply:
initially this was not required on FE side, but at compile time we get a sintax error after using 'yarn generate'.

the generated code contains an error that blocks the compiling of the app, 
if we use required: true, the error does not happen.

Reply:
maybe without             
```
"required": true,
```

it's attempt to "raggirare" codegen :)

Reply:
i will try this on a dummy branch to see if it helps

Reply:
no luck, same result
![image](https://github.com/user-attachments/assets/4cc309cb-0fc1-4d6e-948b-1fde964496a6)

Reply:
this + updating the library on FE did the trick, thanks for the hint

Reply:
didn't work, even after updating the codegen lib :(